pages.de/common/rm.md

```diff
@@ -0,0 +1,24 @@
+# rm
+
+> Lösche Dateien oder Verzeichnisse.
+> Weitere Informationen: <https://www.gnu.org/software/coreutils/rm>.
+
+- Lösche Dateien an beliebigen Speicherorten:
+
+`rm {{pfad/zu/datei}} {{pfad/zu/anderer/datei}}`
+
+- Lösche ein Verzeichnis und alle seine Unterverzeichnisse rekursiv:
+
+`rm -r {{pfad/zu/verzeichnis}}`
+
+- Erzwinge das Löschen eines Verzeichnis, ohne Eingabeaufforderung zur Bestätigung oder Anzeigen von Fehlermeldungen:
+
+`rm -rf {{pfad/zu/verzeichnis}}`
+
+- Lösche mehrere Dateien mit Eingabeaufforderung zur Bestätigung für jede Datei:
+
+`rm -i {{datei(en)}}`
```

Comment:
Why did you mark @EmilySeville7cfg's comment as resolved without explaining or changing anything? We are changing the other versions of this page, what is your rationale for not adopting it here?

```suggestion
`rm -i {{pfad/zu/datei1 pfad/zu/datei2 ...}}`
```

Reply:
When I first saw the comment I could not figure out what was meant, and I did not find a difference in other versions either. I am sorry for not asking.

Reply:
> When I first saw the comment I could not figure out what was meant, and I did not find a difference in other versions either. I am sorry for not asking.

I can mean that I have to describe issues better.